dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.jakarta.concurrency-2.0.feature

```diff
@@ -2,6 +2,7 @@
 symbolicName=io.openliberty.jakarta.concurrency-2.0
 visibility=private
 singleton=true
+-features=com.ibm.websphere.appserver.eeCompatible-9.0
```

Comment:
> Should we make a concurrent-1.0 or concurrency-1.0 API feature with eeCompatible in it and have features depend on it instead of pulling in the bundle directly?

That makes sense.  Should consider in another PR